fabfile.py

```diff
@@ -223,6 +224,10 @@ def install_prometheus(conn):
         'tar xvfz prometheus*.tar.gz'
     conn.run(install_cmd)
 
+@task
+def kill_nodes(conn):
+    kill_cmd = f'killall aleph-node'
+    conn.run(kill_cmd)
```

Comment:
no need for `f` and I would inline the kill_cmd into run function

Reply:
Good point